src/Application.java

```diff
@@ -4,12 +4,12 @@ public static void main(String[] args) {
         var ferrari = new ItalianCar();
         var bg6 = new BritishCar();
         var seat2 = new SpanishCar();
```

Comment:
You have not done what I said.
I remind:
1Create repo 
2 create **an empty** main branch 
3 create a feature branch with your development 
4 create PR for main